src/pages/Common/LoginPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { loggingStateAtom } from 'services/recoil/appState';
+import { useNavigate } from 'react-router-dom';
+import WalletButton from 'components/Solana/SolanaWallet/WalletButton';
+import Spinner from 'components/Util/Spinner';
+import { userInfoAtom } from 'services/recoil/userInfo';
+
+export default function LoginPage(): JSX.Element {
+  const navigate = useNavigate();
+  const loggingState = useRecoilValue(loggingStateAtom);
+  const [logMessage, setLogMessage] = useState('');
+
+  const userInfo = useRecoilValue(userInfoAtom);
+
+  useEffect(() => {
+    switch (loggingState) {
+      case 'receive':
+        setLogMessage('Waiting for message');
+        break;
+      case 'sign':
+        setLogMessage('Please sign the message');
+        break;
+      case 'verify':
+        setLogMessage('Verifying the signature');
+        break;
+      case 'authed':
+        setLogMessage('Welcome !');
+        break;
+      default:
+        setLogMessage('');
+        break;
+    }
+  }, [loggingState]);
+
+  useEffect(() => {
+    if (userInfo !== undefined) {
+      navigate('/');
+    }
+  }, [userInfo]);
+
+  return (
+    <div className="min-h-full page-frame">
+      <main
+        className="flex-grow flex flex-col 
+      justify-center 
+       w-full mx-auto mt-8">
+        <h1
+          className="mt-4 text-xl text-center font-extrabold 
+          text-color-0 leading-tight">
+          Sign in to your account <br /> or create a new one.
+        </h1>
+        <p
+          className="mt-1 text-base 
+        font-semibold text-center text-color-secondary max-w-md mx-auto">
+          Welcome !
+        </p>
+        <div className="w-64 mx-auto mt-2">
```

Comment:
Instead of using fixed widths, like w-64, I generally prefer relative widths like w-1/3 since these widths are more responsive and avoid weird cases for weird screen sizes.